### PR TITLE
Release package

### PR DESCRIPTION
**Pull request by gbhoot**

### CODE REVIEW

File: openapi/v2/change/release_package.yml [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ReleasePackage'
+        '404':
+          description: Change Order not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  '/bom/{itemNumber}':
```
Comment (pkjayk) [inline openapi/v2/change/release_package.yml]: Can this be in a separate file?

File: openapi/v2/change/release_package.yml [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+          default: false
+      - name: limit
+        in: query
+        description: Used to limit the number of affected items included in the response
+        required: false
+        schema:
+          type: integer
+          format: integer
+          default: 0
+      - name: page
```
Comment (pkjayk) [inline openapi/v2/change/release_package.yml:85]: If I make the first call with limit=100, wouldn't the next page need params of `limit=100&page=2`.

Is this much different than `limit=100&offset=100`?

File: openapi/v2/change/release_package.yml [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+            $ref: '#/components/schemas/Assembly'
+        aml:
+          type: array
+          items:
+            $ref: '#/components/schemas/Manufacturer'
+        attachments:
+          type: array
+          items:
+            $ref: '#/components/schemas/Document'
+    Assembly:
```
Comment (pkjayk) [inline openapi/v2/change/release_package.yml:168]: Do you think we should label schema by their API Name? 

So,
```
AffectedItem: Affected_Item__c
Assembly: Assembly_c
etc...
```

File: openapi/v2/change/release_package.yml [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+        PDLM__Item_Number__c:
+          type: string
+          description: Name of the child Item record
+        PDLM__Level__c:
+          type: string
+          description: 'BOM level of assembly row. In the format of 1, 1.1, 2, 2.1, 2.2, 2.2.1'
+        Custom_Field__c:
+          type: string
+          description: Placeholder for any custom fields an org may have on the Assembly__c object.
+    Attachment:
```
Comment (pkjayk) [inline openapi/v2/change/release_package.yml:188]: is this just ContentDocument?

**Review by pkjayk — COMMENTED** [review]
I think we should review how we structure the spec (multiple files vs a single file) and name Schema.

Otherwise, this is a good start! 💯 

In a future PR, let's separate all the schema, endpoints, responses into separate files and bundle them using an available library

**Comment by gbhoot** [comment]
> I think we should review how we structure the spec (multiple files vs a single file) and name Schema.
> 
> Otherwise, this is a good start! 💯
> 
> In a future PR, let's separate all the schema, endpoints, responses into separate files and bundle them using an available library

File: openapi/v2/change/release_package.yml (hunk 4 above)
Comment (gbhoot) [reply openapi/v2/change/release_package.yml:188]: Its a content version record. figured the best way to get the content version Id, plus the contentDocumentId.

File: openapi/v2/change/release_package.yml (hunk 1 above)
Comment (gbhoot) [reply openapi/v2/change/release_package.yml]: Yeah will be. I figured better in a next PR. This file translates well to postman/swagger as an all-in-one file for right now. I havent tested the modularized format and how that connects to a postman yet.

File: openapi/v2/change/release_package.yml (hunk 2 above)
Comment (gbhoot) [reply openapi/v2/change/release_package.yml:85]: Same thing, we can use either. whichever way, we will generate what the 2nd,3rd,... url will be in the 'next' parameter response. Whichever param you think is better.

**Pull request merged**